python/cugraph_service/cugraph_service_server/cugraph_handler.py

```diff
@@ -441,6 +442,9 @@ def load_csv_as_edge_data(
         if len(names) == 0:
             names = None
 
+        if edge_id_col_name == "":
+            edge_id_col_name = None
```

Comment:
Why this change in default ?

Reply:
So that an empty string is not passed to `add_edge_data`, which will result in an error when the expected behavior is to auto-generate the ids.  The correct interpretation of "" received from the client is None, since "" is used as a substitute for None as `edge_id_col_name` has to be a string.